app/src/main/java/com/vitorpamplona/amethyst/ui/navigation/DrawerContent.kt

```diff
@@ -206,66 +214,84 @@ fun ListContent(
 
-    Column(modifier = modifier) {
-        LazyColumn() {
-            item {
-                if (accountUser != null)
-                    NavigationRow(navController,
-                        scaffoldState,
-                        "User/${accountUser.pubkeyHex}",
-                        Route.Profile.icon,
-                        stringResource(R.string.profile)
-                    )
+    Column(modifier = modifier.fillMaxHeight()) {
+        if (accountUser != null)
+            NavigationRow(
+                title = stringResource(R.string.profile),
+                icon = Route.Profile.icon,
+                tint = MaterialTheme.colors.primary,
+                navController = navController,
+                scaffoldState = scaffoldState,
+                route = "User/${accountUser.pubkeyHex}",
+            )
 
-                Divider(
-                    modifier = Modifier.padding(bottom = 15.dp),
-                    thickness = 0.25.dp
-                )
-                Column(modifier = modifier.padding(horizontal = 25.dp)) {
-                    Row(modifier = Modifier.clickable(onClick = {
-                        navController.navigate(Route.Filters.route)
-                        coroutineScope.launch {
-                            scaffoldState.drawerState.close()
-                        }
-                    })) {
-                        Text(
-                            text = stringResource(R.string.security_filters),
-                            fontSize = 18.sp,
-                            fontWeight = W500
-                        )
-                    }
-                    Row(modifier = Modifier.clickable(onClick = { accountViewModel.logOff() })) {
-                        Text(
-                            text = stringResource(R.string.log_out),
-                            modifier = Modifier.padding(vertical = 15.dp),
-                            fontSize = 18.sp,
-                            fontWeight = W500
-                        )
-                    }
-                }
-            }
-        }
+        Divider(thickness = 0.25.dp)
+
+        NavigationRow(
+            title = stringResource(R.string.security_filters),
+            icon = Route.Filters.icon,
+            tint = MaterialTheme.colors.onBackground,
+            navController = navController,
+            scaffoldState = scaffoldState,
+            route = Route.Filters.route,
+        )
+
+        Divider(thickness = 0.25.dp)
+
+        IconRow(
+            title = "Backup Keys",
+            icon = R.drawable.ic_key,
+            tint = MaterialTheme.colors.onBackground,
+            onClick = { backupDialogOpen = true }
+        )
+
+        Spacer(modifier = Modifier.weight(1f))
+
+        IconRow(
+            "Logout",
+            R.drawable.ic_logout,
+            MaterialTheme.colors.onBackground,
+            onClick = { accountViewModel.logOff() }
+        )
```

Comment:
The `Spacer` with `weight(1f)` causes the items below to go to the bottom of the `Column`. Moved the `Logout` button here.